addons/website_sale_product_configurator/views/website_sale_product_configurator_templates.xml

```diff
@@ -4,6 +4,21 @@
         <xpath expr="//th[hasclass('td-qty')]/span[hasclass('label')]" position="attributes">
             <attribute name='t-if'>not request.is_frontend or (request.is_frontend and is_view_active('website_sale.product_quantity'))</attribute>
         </xpath>
+        <xpath expr="//td[hasclass('td-price')]" position="inside">
+            <div
```

Comment:
If we set `compare_list_price` to some value greater than `price`, and then we deactivate the comparison price in the settings, the striked price is still shown. Could you do something like [1] to prevent this?

[1] https://github.com/odoo/odoo/blob/acddba7b23c7f1fa581499d368daa272eda14bfc/addons/website_sale/views/templates.xml#L1178

Reply:
Done too :+1:  Got to learn something new here :see_no_evil: